lib/player.rb

```diff
@@ -0,0 +1,33 @@
+# rubocop:disable Style/ClassVars
+class Player
+  @@players_arry = []
+  @@current_player = true
+  attr_accessor :name
```

Comment:
- [ ] You should use  attr_readers or attr_writers. Only using attr_accessors it allows writing that property from outside the class. That variable can be set to anything. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))